public/index.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Firefox Fileshare</title>
-<script src="upload.js"></script>
+  <title>Firefox Fileshare</title>
+  <script src="bundle.js"></script>
```

Comment:
Not sure if we want this as "bundle.js" or "/bundle.js" (to be consistent w/ _public/download.html_ above)...